x-pack/plugins/security_solution/public/common/components/endpoint/host_isolation/isolate_success.tsx

```diff
@@ -7,27 +7,43 @@
 
 import React, { memo, ReactNode } from 'react';
 import { EuiButtonEmpty, EuiCallOut, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
-import { GET_SUCCESS_MESSAGE } from './translations';
+import { GET_ISOLATION_SUCCESS_MESSAGE, GET_UNISOLATION_SUCCESS_MESSAGE } from './translations';
 
 export interface EndpointIsolateSuccessProps {
   hostName: string;
+  isolateAction?: string;
```

Comment:
Did you mean for this to be any `string` or something like:

```suggestion
  isolateAction?: 'isolate` | `unisolate`
```